### ISSUE TITLE
Perl - endpwent

### ISSUE THREAD

**Issue by fabioyy** [termux/termux-packages#212]
Perl  is not working here. ( pixel-c )

$ perl
CANNOT LINK EXECUTABLE: cannot locate symbol "endpwent" referenced by "perl"...
Aborted (core dumped)

isnt endpwent removed from termux libc?

**Comment by fornwall** [comment]
The `endpwent()` function is a stub in the bionic libc for devices I have access to, perhaps it has finally been removed entirely on the pixel c.

The function has been patched away for newer builds of termux packages. Can you try the below package and see if it works better?

    wget http://fornwall.net/perl_5.22.1_aarch64.deb
    dpkg -i perl_5.22.1_aarch64.deb

**Comment by fabioyy** [comment]
yep, it worked! thank you!!!!

( ps: i' m running on android N preview )

**Comment by fornwall** [comment]
@fabioyy Thanks for the feedback! I've rebuilt perl (new version: `5.22.1-1`) and pushed the rebuilt package to the apt repository, so it should be fixed now.

**Comment by tonyseek** [comment]
There is a similar issue for `find` command.

```
$ find
CANNOT LINK EXECUTABLE "find": cannot locate symbol "endpwent" referenced by "/data/data/com.termux/files/usr/bin/find"...
[1]    14338 abort      find
```

**Comment by fornwall** [comment]
@tonyseek Thanks - this should now be fixed in the `4.6.0-1` update of the findutils package!
